Wikipedia/Code/TalkPageLocalHandler.swift

```diff
@@ -124,7 +135,7 @@ extension NSManagedObjectContext {
     func removeUnlinkedTalkPageTopicContent() throws {
         let request: NSFetchRequest<TalkPageTopicContent> = TalkPageTopicContent.fetchRequest()
         request.predicate = NSPredicate(format: "topics.@count == 0")
-        let results = try request.execute()
+        let results = try fetch(request)
```

Comment:
before this change I was getting a "Cannot fetch without an NSManagedObjectContext in scope}" console error and TalkPageTopicContent objects were not clearing out in housekeeping.